api/models/OfficerManager.js

```diff
@@ -17,29 +14,19 @@ const OfficerSchema = new Schema(
     github: {
       type: String
     },
-    instagram: {
+    linkedin: {
       type: String
     },
-    // The access level is defined as follows:
-    // -2: Ban
-    // -1: Pending
-    // 0: Member
-    // 1: Officer
-    // 2: Admin
-    level: {
-      type: Number,
-      require: true
-    },
     team: {
       type: String,
       require: true
     },
     quote: {
       type: String
     },
-    major: {
+    picture: {
```

Comment:
`picture` is a bit ambigious, and in the future it may improve readability if we do
```suggestion
    pictureUrl: {
```

```jsx

// pretend we are rendering an array of officers
{officers.map((officer, index) => {
  return (
    <div>
      {/* react stuff here */}
      <Image src={officer.pictureUrl} alt='officer profile' />
    </div>
  )
})}
```